app/src/main/java/org/luxoft/sdl_core/BluetoothHandler.java

```diff
@@ -120,6 +125,9 @@ public void run() {
                     }
                 }, 5000);
             }
+
+            final Intent intent = new Intent(ON_BLE_PERIPHERAL_DISCONNECT);
+            context.sendBroadcast(intent);
```

Comment:
@VadymLuchko can look like that after the proposed changes:
```suggestion
            final Intent intent = new Intent(ON_BLE_PERIPHERAL_DISCONNECT);
            final byte[] message = GenerateDisconnectMessage(); // TODO
            intent.putExtra(BleCentralService.MOBILE_CONTROL_DATA_EXTRA, message);
            context.sendBroadcast(intent);
```

Reply:
done in 66d56ca414ac2d17dea823a2186a8d3f2569ec83